README.md

```diff
@@ -22,7 +22,7 @@ The OLS Governance Committee (referred to as ‘Committee’ hereafter) is a ded
 * Hao Ye, University of Pennsylvania / Community for Rigor, USA
 * Laura Ación, MetaDocencia, Argentina 
 * Laurah Ondari, Bioinformatics Hub of Kenya initiative (BHKi), Kenya
-* Rowland Mosbergen, Practical Diversity and Inclusion Project, New Zealand 
+* Rowland Mosbergen, Practical Diversity and Inclusion Project, Australia
```

Comment:
Grave mistake! Spotted by a Kiwi.

Reply:
I knew something was fishy. 👀 I have to quickly change it in the main governance page.

Reply:
Malvika's basically saying she prefers ghanain jollof rice here...